client/client.go

```diff
@@ -403,6 +404,38 @@ func (w *WrapperV2) GetCluster(clusterID string, p *AuxiliaryParams) (*clusters.
 	return response, nil
 }
 
+// GetClusterV5 fetches details on a V5 cluster.
+func (w *WrapperV2) GetClusterV5(clusterID string, p *AuxiliaryParams) (*clusters.GetClusterV5OK, error) {
```

Comment:
Would it make more sense to have a package separation here like we do in operators instead of renaming our functions?

Reply:
Not sure, as I don't develop on operators